filters/builtin/mod_header.go

```diff
@@ -0,0 +1,79 @@
+package builtin
+
+import (
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/zalando/skipper/filters"
+)
+
+type modRequestHeader struct {
+	headerName  string
+	rx          *regexp.Regexp
+	replacement string
+}
+
+// Returns a new modRequestHeader filter Spec, whose instances execute
```

Comment:
Go doc wants that you start all doc comments by the function name.
So:

NewModRequestHeader returns a new filter Spec, whose instances execute...

Reply:
Done